osbs/core.py

```diff
@@ -93,6 +92,7 @@ def __init__(self, openshift_api_url, openshift_api_version, openshift_oauth_url
         self.verbose = verbose
         self.verify_ssl = verify_ssl
         self._con = HttpSession(verbose=self.verbose)
+        self.retries_disabled = False
```

Comment:
It's often better to name things the other way around, e.g. `retries_enabled`, otherwise when reading code you end up trying to work out "double negatives" (e.g. "not disabled").